openpype/hosts/maya/plugins/publish/validate_render_single_camera.py

```diff
@@ -47,7 +38,9 @@ def get_invalid(cls, instance):
         # handle various renderman names
         if renderer.startswith('renderman'):
             renderer = 'renderman'
-        file_prefix = cmds.getAttr(ImagePrefixes[renderer])
+
+        attr = RenderSettings.get_image_prefix_attr(renderer)
+        file_prefix = cmds.getAttr(attr)
```

Comment:
Shouldn't we just be able to do `RenderSettings.get_image_prefix(layer)` as a single call to make this usage as simple as possible?

Reply:
this is directly connected to collectors and the issue we've hit on scene with hundreds of render layers, where these calls are causing terrible performance issues

Reply:
The current code doesn't really solve that, right? Or how does this differ so that it's much faster?

Reply:
no, that was just a note, it doesn't solve it